packages/ui/src/composites/wui-button/styles.ts

```diff
@@ -7,60 +7,141 @@ export default css`
-    padding: var(--wui-spacing-xs) var(--wui-spacing-1xs);
+  button[data-size='md'][data-icon-left='true'][data-icon-right='false'] {
+    padding: 8.2px var(--wui-spacing-l) 9px var(--wui-spacing-s);
   }
 
-  button[data-size='xs'][data-icon-left='true'][data-icon-right='false'] {
-    padding-left: var(--wui-spacing-xxs);
+  button[data-size='md'][data-icon-right='true'][data-icon-left='false'] {
+    padding: 8.2px var(--wui-spacing-s) 9px var(--wui-spacing-l);
   }
 
-  button[data-size='xs'][data-icon-right='true'][data-icon-left='false'] {
-    padding-right: var(--wui-spacing-xxs);
+  button[data-size='lg'] {
+    padding: var(--wui-spacing-m) var(--wui-spacing-2l);
+    height: 48px;
   }
 
-  /* sm settings */
-  button[data-size='sm'] {
-    padding: var(--wui-spacing-xxs) var(--wui-spacing-s);
+  /* -- Variants --------------------------------------------------------- */
+  button[data-variant='main'] {
+    background-color: var(--wui-color-accent-100);
+    color: var(--wui-color-inverse-100);
+    border: none;
+    box-shadow: inset 0 0 0 1px var(--wui-color-gray-glass-010);
   }
 
-  button[data-size='sm'][data-icon-left='true'][data-icon-right='false'] {
-    padding: var(--wui-spacing-xxs) var(--wui-spacing-s) var(--wui-spacing-xxs)
-      var(--wui-spacing-xs);
+  button[data-variant='inverse'] {
+    background-color: var(--wui-color-inverse-100);
+    color: var(--wui-color-inverse-000);
+    border: none;
+    box-shadow: inset 0 0 0 1px var(--wui-color-gray-glass-010);
   }
 
-  button[data-size='sm'][data-icon-right='true'][data-icon-left='false'] {
-    padding: var(--wui-spacing-xxs) var(--wui-spacing-xs) var(--wui-spacing-xxs)
-      var(--wui-spacing-s);
+  button[data-variant='accent'] {
+    background-color: var(--wui-color-accent-glass-010);
+    color: var(--wui-color-accent-100);
+    border: none;
+    box-shadow: inset 0 0 0 1px var(--wui-color-gray-glass-005);
   }
 
-  /* md settings */
-  button[data-size='md'] {
-    padding: 8.2px var(--wui-spacing-l) 9px var(--wui-spacing-l);
+  button[data-variant='accent-error'] {
+    background: var(--wui-color-error-glass-015);
+    color: var(--wui-color-error-100);
+    border: none;
+    box-shadow: inset 0 0 0 1px var(--wui-color-error-glass-010);
   }
 
-  button[data-size='md'][data-icon-left='true'][data-icon-right='false'] {
-    padding: 8.2px var(--wui-spacing-l) 9px var(--wui-spacing-s);
+  button[data-variant='accent-success'] {
+    background: var(--wui-color-success-glass-015);
+    color: var(--wui-color-success-100);
+    border: none;
+    box-shadow: inset 0 0 0 1px var(--wui-color-success-glass-010);
   }
 
-  button[data-size='md'][data-icon-right='true'][data-icon-left='false'] {
-    padding: 8.2px var(--wui-spacing-s) 9px var(--wui-spacing-l);
+  button[data-variant='neutral'] {
+    background: transparent;
+    color: var(--wui-color-fg-100);
+    border: none;
+    box-shadow: inset 0 0 0 1px var(--wui-color-gray-glass-005);
```

Comment:
aren't  `box-shadow`s abstracted on a design-level? 
Would be nice to not repeat this (and other shadows) all over the place

Reply:
Yeap, the `w3m-input-token` doesn't getting children

Reply:
`box-shadow` is used for border functionality since we don't have inset borders and different variants have different shadow (border) colors. Moving them to a single place would be nice tbh. I'd say let's do that improvement in another PR